core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/hybrid/HybridRoleManager.java

```diff
@@ -1391,13 +1431,17 @@ public boolean isGroupAssignedToHybridRoles(String groupName) throws UserStoreEx
      */
     public void updateGroupName(String groupName, String newGroupName) throws UserStoreException {
 
-        if (!this.isGroupAssignedToHybridRoles(groupName)) {
```

Comment:
Are there any other usages for the method `isGroupAssignedToHybridRoles(String groupName)`? If not shouldn't we deprecate it?
If it is used, why doesn't this problem is not visible there?

Reply:
The existing method is deprecated and a new method is overloaded. 
https://github.com/wso2/carbon-kernel/pull/3713/files#diff-f6ab77b823d681c42be1e812cb3a94308276ef2f08a8407e87de496bbb84e74cR1357